src/docs/software/using/anaconda.md

```diff
@@ -75,6 +75,11 @@ modules on Sherlock,
 And that's it: your software should run, without Anaconda. If you have any
 issues, please don't hesitate to [contact us][url_contact].
 
+!!! Warning "Install anaconda/miniconda in an sh_dev session"
```

Comment:
That section is actually about using Python virtual envs _instead_ of installing Conda, so the warning about installing Conda seems a bit unexpected here.